app/components/object-detail-widget/component.js

```diff
@@ -15,6 +15,25 @@ export default Ember.Component.extend({
         return JSON.stringify(data, null, '    ');
     }),
 
+    identifiers: Ember.computed('objectData._source.identifiers', function() {
+        var data = this.get('objectData');
+        var identifiers = data._source.identifiers;
+        var linkIds = [];
+        var otherIds = [];
+        identifiers.forEach(function(id) {
+            if (id.includes('http')) {
```

Comment:
Ah okay...how about `id.includes('http://')` or should I be more specific and do a regex?